### PR TITLE
Merge dev branch into main

### PR DESCRIPTION
**Pull request by mathisve**
Here is a live example of it working:
https://github.com/Mathisco-01/ghub-actions-test/runs/2051375519?check_suite_focus=true

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	}
+
+	f := drive.File{
+		Name: file.Name(),
+		Parents: []string{folderId},
+	}
+
+	_, err = svc.Files.Create(&f).Media(file).Do()
+	if err != nil {
+		githubactions.Fatalf(fmt.Sprintf("Creating file: %+v failed with error: %v", f, err))
```
Comment (Azer0s) [inline main.go]: Super small nitpick: Creating should be lower case to be consistent with all your other log messages

File: main.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	if err != nil {
+		log.Println(err)
+	}
+
+	file, err := os.Open(filename)
+	if err != nil {
+		githubactions.Fatalf(fmt.Sprintf("opening file with filename: %v failed with error: %v", filename, err))
+	}
+
+	f := drive.File{
```
Comment (Azer0s) [inline main.go]: Please make this a pointer here instead of creating a reference below

File: main.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
+// Mathis Van Eetvelde
+// Ghent 2021
+
+package main
```
Comment (Azer0s) [inline main.go:4]: Please run go fmt and goreportcard locally

**Review by Azer0s — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (mathisve) [reply main.go]: Good catch

File: main.go (hunk 2 above)
Comment (mathisve) [reply main.go]: Done

File: main.go (hunk 3 above)
Comment (mathisve) [reply main.go:4]:
```
Grade: A+ (94.1%)
Files: 1
Issues: 1
gofmt: 100%
go_vet: 100%
gocyclo: 100%
golint: 100%
ineffassign: 100%
license: 0%
misspell: 100%
```
Check slack for the license problem

**Comment by mathisve** [comment]
1898b4 This was an accident, didn't pull the updated README.md before pushing. Oups

File: main.go (hunk 3 above)
Comment (Azer0s) [reply main.go:4]: Replied on slack. Please check there.

**Review by Azer0s — APPROVED** [review]

**Comment by Azer0s** [comment]
Good job.

**Pull request merged**